packages/serverless-contracts/src/types/http.ts

```diff
@@ -1 +1,347 @@
+/* eslint-disable max-lines */
 export type HttpMethod = 'GET' | 'POST' | 'PUT' | 'PATCH' | 'DELETE';
+
+export enum StatusCodes {
```

Comment:
As the library does not only export tools related to HTTP, we should name it `HttpStatusCodes` to be really explicit.